packages/flutter_test/lib/src/matchers.dart

```diff
@@ -354,6 +354,57 @@ Matcher coversSameAreaAs(Path expectedPath, { required Rect areaToCompare, int s
 /// ```
 /// {@end-tool}
 ///
+/// Fonts can be rendered differently according with the platform or the Flutter
+/// version in which the test is run. While the difference may be minimal, it's
+/// still valuable to make the test fail. For example, a golden file generated
+/// on Windows is going to differ from the one produced by another operating
+/// system. Even on the same platform, if the generated golden is tested with a
+/// different Flutter version, the test may fail.
+///
+/// By default, the Flutter framework uses a font called 'Ahem' which shows
+/// squares instead of characters. To make golden tests more valuable, there
+/// is the possibility to render images using other more human-readable fonts.
+/// For example, this is how to load the 'Roboto' font for a golden test:
+///
+/// {@tool snippet}
+/// Using custom fonts in golden images
```

Comment:
This should use full sentence structure.

Reply:
What do you mean by "full sentenze structure"? Are you talking about the line after `{@tool snippet}`?

Reply:
> Are you talking about the line after {@tool snippet}?

I am. And I mean that it should have proper grammar, it is not currently a full sentence. :)